server.py

```diff
@@ -58,6 +61,9 @@ def countTagRoute():
     if url is None:
         return config.CANNOT_FIND_URL_MESSAGE, 404
 
+    if not utils.checkURLWhitelist(url):
+        return config.FORBIDDEN_URL_MESSAGE
+
```

Comment:
These have been added to the `count` routes, it would probably make sense to also add them to the `nocount` routes.